locales/base/translation.json

```diff
@@ -2638,13 +2638,35 @@
       "depositAndEarnings": "Deposit & Earnings"
     },
     "poolFilters": {
-      "myPools": "My Pools",
-      "allPools": "All Pools"
+      "allPools": "All Pools",
+      "myPools": "My Pools"
+    },
+    "poolInfoScreen": {
+      "chainName": "Chain: <0>{{networkName}}</0>",
+      "protocolName": "Protocol: <0>{{providerName}}</0>",
+      "yieldRate": "Yield Rate",
+      "ratePercent": "{{rate}}%",
+      "rewards": "Rewards",
+      "noRewards": "No rewards",
+      "tvl": "TVL",
+      "ageOfPool": "Age of Pool",
+      "learnMoreOnProvider": "View Pool on {{providerName}}",
+      "deposit": "Deposit",
+      "withdraw": "Withdraw"
     }
   },
   "addFundsActions": {
     "add": "Buy",
     "transfer": "Transfer",
     "swap": "Swap"
-  }
+  },
+  "time_lessThanADay": "Less than a day",
```

Comment:
nit: maybe duration over time to stay consistent?

Reply:
Changed in https://github.com/valora-inc/wallet/commit/dc864368ba0cc26257fa6f81294e01792e532566!